docs/guide/cache/content_aware_cache.md

```diff
@@ -474,7 +474,7 @@ The output for this command should look similar to this:
     Vary: cookie
     Vary: authorization
     X-Cache-Debug: 1
-    Surrogate-Key: ez-user-context-hash ez-all fos_http_cache_hashlookup-
+    Surrogate-Key: ibx-user-context-hash ez-all fos_http_cache_hashlookup-
```

Comment:
The same comment as above applies to `ez-user-context-hash`, it was left untouched.